### ISSUE TITLE
(jaana): The current sigmoid_cross_entropy_with_logits has 

### ISSUE THREAD

**Issue by piascikj** [imdone/tensorflow#3421]
inconsistent behavior for logits = inf/-inf.

----
*opened via [imdone.io](https://imdone.io) from a code comment on [a16a92f104fea20da6c0a62ebe2d91ab03429e29](https://github.com/imdone/tensorflow/commit/a16a92f104fea20da6c0a62ebe2d91ab03429e29) by Eugene Brevdo ebrevdo@google.com*

----

https://github.com/imdone/tensorflow/blob/8cb6e535f6ea14b380aa86d425169adad682cb8c/tensorflow/python/ops/distributions/bernoulli.py#L128-L134
